packages/api/test/e2e/governance.e2e.ts

```diff
@@ -0,0 +1,168 @@
+// limitations under the License.
+
+import { Keyring } from '@polkadot/keyring';
+import { cryptoWaitReady } from '@polkadot/util-crypto';
+import { u8aToString } from '@polkadot/util'
+
+import initApiPromise from '../../../../jest/initApiPromise';
+import { AccountId, Vec } from '@cennznet/types';
+import {DeriveProposalInfo} from "@cennznet/api/derives/governance/types";
+
+let api;
+const keyring = new Keyring({ type: 'sr25519' });
+let councilMemberAlice, councilMemberBob, councilMemberCharlie, councilMemberDave;
+
+beforeAll(async done => {
+  await cryptoWaitReady();
+  api = await initApiPromise();
+  const sudoKey = await api.query.sudo.key();
+  // alice is sudo
+  councilMemberAlice = keyring.addFromUri('//Alice');
+  councilMemberDave = keyring.addFromUri('//Dave');
+  councilMemberBob = keyring.addFromUri('//Bob');
+  councilMemberCharlie = keyring.addFromUri('//Charlie');
+  // Lookup from keyring (assuming we have added all, on --dev this would be `//Alice`)
+  const sudoPair = keyring.getPair(sudoKey.toString());
+
+  const transaction1 = api.tx.governance.addCouncilMember(councilMemberAlice.address);
+  const transaction2 = api.tx.governance.addCouncilMember(councilMemberBob.address);
+  const transaction3 = api.tx.governance.addCouncilMember(councilMemberCharlie.address);
+  const transaction4 = api.tx.governance.addCouncilMember(councilMemberDave.address);
+
+  const batchCouncilEx = api.tx.utility.batch([
+    transaction1,
+    transaction2,
+    transaction3,
+    transaction4
+  ]);
+
+  await api.tx.sudo
+    .sudo(batchCouncilEx)
+    .signAndSend(sudoPair, async ({ events, status }) => {
+      if (status.isInBlock) {
+        for (const {event: {method, section, data}} of events) {
+          console.log('Method:', method.toString());
+          console.log('section:', section.toString());
+        }
+        console.log(`Transaction included at blockHash ${status.asInBlock}`);
+        done();
+      }
+    });
+
+});
+
+afterAll(async () => {
+  await api.disconnect();
+});
+
+describe('Governance', () => {
+
+  it('List all council members', async done => {
+    const councilMembers: Vec<AccountId> = await api.query.governance.council();
+    console.log('Councilmemmbers::....', councilMembers.toJSON());
+    const members: string[] = councilMembers.toJSON() as unknown as string[];
+    expect(members.includes(councilMemberBob.address)).toBeTruthy();
+    expect(members.includes(councilMemberAlice.address)).toBeTruthy();
+    expect(members.includes(councilMemberCharlie.address)).toBeTruthy();
+    expect(members.includes(councilMemberDave.address)).toBeTruthy();
+    done();
+  });
+
+  it('creates a proposal to add new council member', async done => {
+    const newCouncilMember = '5FWEHQqYMN8YCg8yJxKHnon7Dtx4Psp2xnjvKfQqGC6kUwgv';
+    const call = api.tx.governance.addCouncilMember(newCouncilMember);
+    const proposalCall = api.registry.createType('Call', call).toHex();
```

Comment:
out of interest, does this work too?
```suggestion
    const proposalCall = api.tx.governance.addCouncilMember(newCouncilMember).toHex();
```

Reply:
no.. it didn't work.. generates a different hex